tutorials/script/mk-tutorial.py

```diff
@@ -133,29 +147,14 @@ def stepsStr(i):
                 out.write(f'\n```sh\n')
                 out.write(f'{c.text}\n')
                 out.write(f'```\n\n')
-
-        # insert a piece of code
-        line = step.find("line")
-        line = int(line.text) if line != None else None
-        # the first step is normally an intro, its code will be skipped
-        if line != None and lastLine != None:
-            out.write('**Example:**\n\n')
-            out.write('```scala\n')
-            for l in code[int(lastLine) + 1: int(line)]:
-                out.write(l)
-
-            out.write('```\n\n')
           
         # update the last line, so it can be used in the next iteration
         lastLine = line
 
-    out.write("# Done!\n")
+    out.write("## The end\n")
```

Comment:
Should we write the link to the feedback discussion to all tutorials?

>We are experimenting with different kinds of tutorials.
It would be great to learn, whether you liked this tutorial format, or not.
Please vote in the
[discussion](https://github.com/informalsystems/quint/discussions/509).

Reply:
I would rather introduce a discussion for every tutorial separately. It may happen that something specific to a tutorial is affecting the answers. If we have different discussions, we would be able to see that there is another hidden variable in the tutorials.